x/poe/contract/tg4_engagement.go

```diff
@@ -46,6 +47,12 @@ type UpdateMembersMsg struct {
 	Remove []string    `json:"remove"`
 }
 
+// WithdrawRewardsMsg contract execute message to claim rewards
+type WithdrawRewardsMsg struct {
+	Owner    *string `json:"owner,omitempty"`
+	Receiver *string `json:"receiver,omitempty"`
```

Comment:
At some point it would be nice to add support for these two options as well.

Reply:
Created #427